std_msgs/Header header
sensor_msgs/Image left_image
sensor_msgs/Image right_image
